definitions/oi-kamon-agent/definition.yml

```
@@ -0,0 +1,18 @@
+domain: EXT
+type: KAMON_AGENT
```

Just to start conversation, would this make sense? (mimics our own jvm agent apm|application and browser|application)

```suggestion
domain: KAMON
type: APPLICATION
```